R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -3633,7 +3633,8 @@ test_that("catalog APIs, currentDatabase, setCurrentDatabase, listDatabases", {
   expect_equal(currentDatabase(), "default")
   expect_error(setCurrentDatabase("default"), NA)
   expect_error(setCurrentDatabase("zxwtyswklpf"),
-        "Error in setCurrentDatabase : analysis error - Database 'zxwtyswklpf' does not exist")
+               paste("Error in setCurrentDatabase : analysis error - Database",
```

Comment:
@HyukjinKwon here i am fixing the last un-addressed [comment](https://github.com/apache/spark/pull/22161#discussion_r212168906)

Reply:
I'd use paste0 instead to make clear about the implicit space that should be after `Database`

ie. `paste0("Error in setCurrentDatabase : analysis error - Database ",  "'zxwtyswklpf' does not exist"))

Reply:
@felixcheung Sure.

Reply:
@felixcheung Made the change to use paste0. Please take a look when you get a chance. Thanks !!